docs-web/configuration/security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+We consider the security of the Ingress Controller paramount to the success of our users and use the following to 
+ensure the security of the Ingress Controller:
+* [Secure Development Life Cycle](https://www.microsoft.com/en-us/securityengineering/sdl/)
+* [FOSSA](https://fossa.com) scanning
+
+However, the Ingress Controller is deployed by a User in their environment, and as such, the User takes responsibility 
+for securing a *deployment* of the Ingress Controller. 
+We highly recommend every User to read and understand the following security concerns.
+
+## Kubernetes
+We recommend the Kubernetes [guide to securing a cluster](https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/).
+In addition, the following relating more specifically to Ingress Controller.
+
+### RBAC and Service Account
+The Ingress Controller is deployed within a Kubernetes environment, this environment must be secured. 
+Kubernetes uses [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to control which types of users 
+can access which resources and what operations they can perform. 
```

Comment:
Perhaps:
"Kubernetes uses [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to control the resources and operations available to different types of users."

Reply:
sure